src/locales/pl.json

```diff
@@ -304,5 +306,8 @@
       "title": "Publiczna"
     },
     "title": "Widoczność tablicy"
+  },
+  "all-events": {
+    "title": "Wszystkie wydarzenia"
   }
```

Comment:
Please sort properties alphabetically (action available in WebStorm quick action menu)